cloudferrylib/os/actions/check_needed_compute_resources.py

```diff
@@ -50,3 +54,18 @@ def check(name, units, have, needed):
                                            "Have %s %s, needed %s %s." % (
                                                name, have, units, needed,
                                                units))
+
+    def check_in_use_flavor(self, objs):
+        # when a flavor is updated, the flavor id will change. If an instance
+        # is in this flavor, it will keep the old flavor id, can not be matched
+        # to existing flavors
+        src_compute = self.src_cloud.resources[utl.COMPUTE_RESOURCE]
+        src_flavors = [flavor.id for flavor in src_compute.get_flavor_list()]
+
+        for instance in objs.values():
+            if instance['instance']['flavor_id'] not in src_flavors:
+                LOG.info("The flavor ID of instance %s does not match to any"
+                         " flavor. Pleas resize it.",
```

Comment:
Ok, I've got it. Thanks. 
Just 3 small nits here:
1. Typo in word 'please'
2. I guess we should use another log level for such case (Critical or Error)
3. Could you change message 'Pleas resize it' to 'Please resize the VM', cause f.e. it was not so obvious for me.